xcode/Subconscious/Shared/Models/Petname.swift

```diff
@@ -73,5 +73,10 @@ public struct Petname:
     public init?(formatting string: String) {
         self.init(Self.format(string))
     }
+    
+    public init?(petnames: [Petname]) {
+        let petnamePath = petnames.map({ s in s.verbatim }).joined(separator: ".")
+        self.init(petnamePath)
+    }
```

Comment:
This will be removed / replaced by the code in https://github.com/subconsciousnetwork/subconscious/pull/489